tcl/wheredoesthisgo.tcl

```diff
@@ -28,28 +28,28 @@ proc open_file {filename} {
 # ------------------------------------------------------------------------------
 # procs for panels (openpanel, savepanel)
 
-proc pdtk_openpanel {target localdir} {
+proc pdtk_openpanel {target localdir parent} {
```

Comment:
probably the `parent` argument should be made optional:

~~~tcl
proc pdtk_openpanel {target localdir {parent .pdwindow}} {
~~~